backoffice/src/config.ts

```diff
@@ -2,5 +2,5 @@ import { serviceUrl } from "./utils/urls";
 
 export const applicationConfig = {
   apiUrl: serviceUrl(window.location.hostname),
-  apiTimeoutMs: 60000,
+  apiTimeoutMs: 120000,
```

Comment:
This was already 60 seconds, which is longer than the 30 seconds we are seeing, so I can't imagine it would have any effect, but happy to try any experiments and would love to be wrong.